### PR TITLE
TELCODOCS-1691, TELCODOCS-1851 - Adding Telco core and Telco RAN 4.15 RDS updates

### PR DESCRIPTION
**Pull request by aireilly**
Adding Telco core and Telco RAN 4.15 RDS updates. YAML updates from:
* `registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.15` (RAN)
* `https://github.com/openshift-kni/telco-reference/tree/release-4.15` (Core)

Version(s):
enterprise-4.15

Issue:
https://issues.redhat.com/browse/TELCODOCS-1691 (Core)
https://issues.redhat.com/browse/TELCODOCS-1851 (RAN)

Link to docs preview:
https://file.emea.redhat.com/aireilly/telco-rds-4.15/telco_ref_design_specs/telco-ref-design-specs-overview.html

QE review:
- [x] QE review not required.

### CODE REVIEW

**Review by imiller0 — COMMENTED** [review]
/lgtm

**Review by kcarmichael08 — COMMENTED** [review]
Great job! I had some minor comments & small issues, otherwise it looks fantastic!

**Comment by kcarmichael08** [comment]
Also don't forget to squash commits, check QA box, etc.

File: modules/telco-core-sriov.adoc [hunk 1]
```diff
@@ -8,17 +8,29 @@
 
 New in this release::
 
-Not applicable
+//CNF-5528
+* `MultiNetworkPolicy` resources can now be applied to SR-IOV networks to enforce network reachability policies.
+
+//CNF-9865
+* QinQ is now supported in the SR-IOV Network Operator.
+This is a Tech Preview feature.
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/telco-core-sriov.adoc:16]: We do use the TP snippets in the core docs, but we don't need it here.

File: modules/telco-ran-yaml-ref-day-2-operators.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * telco_ref_design_specs/ran/telco-ran-ref-du-crs.adoc
+// *
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/telco-ran-yaml-ref-day-2-operators.adoc:3]: Lol it does thanks

**Pull request merged**